testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -55,4 +59,16 @@ public static String getPathFromClassPathResource(String resourcePath)
                 // and mounted from there.
                 .getResolvedPath();
     }
+
+    public static void configureFixedExposedPortForLocalEnv(GenericContainer<?> container)
+    {
+        checkState(System.getenv("CONTINUOUS_INTEGRATION") == null,
+                "Exposing fixed ports should not be used in regular test code. " +
+                        "This could break parallel test execution locally. " +
+                        "This method is supposed to be invoked from local development helpers only, hence it should never be run on CI");
+        for (int exposedPort : container.getExposedPorts()) {
+            container.withCreateContainerCmdModifier(cmd -> cmd.withHostConfig(
+                    new HostConfig().withPortBindings(new PortBinding(bindPort(exposedPort), new ExposedPort(exposedPort)))));
+        }
+    }
```

Comment:
despite this implementation looks laconic, I'd prefer to have it more verbose, but 'dumb' - removing developers need to iterate the final result:
```
        ImmutableList<PortBinding> portBindings = container.getExposedPorts().stream()
                .map(exposedPort -> new PortBinding(bindPort(exposedPort), new ExposedPort(exposedPort)))
                .collect(toImmutableList());
                
        container.withCreateContainerCmdModifier(cmd -> cmd
                .withHostConfig(new HostConfig()
                        .withPortBindings(portBindings)));
```